modules/transpiler/epcis/epcis-otjson-transpiler.js

```diff
@@ -70,21 +71,20 @@ class EpcisOtJsonTranspiler {
         const transpilationInfo = this._getTranspilationInfo();
         transpilationInfo.diff = json;
 
-        otjson['@id'] = importUtilities.calculateGraphPublicHash(otjson['@graph']);
+        otjson['@id'] = '';
         otjson['@type'] = 'Dataset';
-
         otjson.datasetHeader = importUtilities.createDatasetHeader(this.config, transpilationInfo);
 
-        const merkleRoot = importUtilities.calculateDatasetRootHash(otjson['@graph'], otjson['@id'], otjson.datasetHeader.dataCreator);
-
-        otjson.datasetHeader.dataIntegrity.proofs[0].proofValue = merkleRoot;
+        let result = otjson; // todo add otJsonService
```

Comment:
Do we need to call otjson service here before calculating graph public has and root hash?